docs/reference/analysis/tokenfilters/multiplexer-tokenfilter.asciidoc

```diff
@@ -19,7 +19,7 @@ filters:: a list of token filters to apply to incoming tokens.  These can be any
   using a comma-delimited string, so for example `"lowercase, porter_stem"` would
   apply the `lowercase` filter and then the `porter_stem` filter to a single token.
 
-WARNING: Shingle or multi-word synonym token filters will not function normally
+WARNING: Single or multi-word synonym token filters will not function normally
```

Comment:
This is not a typo, but refers to [shingle token filters](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-shingle-tokenfilter.html). Since this has confused you, maybe its not clear enough. What could we do to make it easier to understand?

Reply:
Oh! my false, I'm new to elasticsearch and this word is not familiar to me.

btw, The docs itself is perfectly understandable, but it's would be great to add a hyperlink to refers "shingle token filters" in this context. it might prevent misunderstanding for new coming like me.

Thanks.

Reply:
Great, since you already opened the PR, maybe you'd be interested in making that change and pushing it as an update to this PR? Otherwise we can just close it.